SYCL/ESIMD/regression/atomic_update_test.cpp

```diff
@@ -0,0 +1,46 @@
+//==---------------- atomic_smoke.cpp  - DPC++ ESIMD on-device test --------==//

```

Comment:
Minor comment: it is copy-paste error here. Please change the name of the file in this line.